src/pages/inspect/block/ui/block-summary.tsx

```diff
@@ -0,0 +1,101 @@
+import { InfoCard } from '@/pages/explore/ui/info-card';
+import { Text } from '@penumbra-zone/ui/Text';
+import { Table } from '@penumbra-zone/ui/Table';
+import { BlockSummaryApiResponse } from '@/shared/api/server/block/types';
+import { ValueViewComponent } from '@penumbra-zone/ui/ValueView';
+import { pnum } from '@penumbra-zone/types/pnum';
+
+export function BlockSummary({ blockSummary }: { blockSummary: BlockSummaryApiResponse }) {
+  if ('error' in blockSummary) {
+    return <div>Error: {blockSummary.error}</div>;
+  }
+
+  return (
+    <div>
+      <div className='grid grid-cols-1 tablet:grid-cols-2 desktop:grid-cols-3 gap-2 mb-8'>
+        <InfoCard title='Total Transactions'>
+          <Text large color='text.primary'>
+            {blockSummary.numTxs}
+          </Text>
+        </InfoCard>
+        <InfoCard title='Total Swaps'>
+          <Text large color='text.primary'>
+            {blockSummary.numSwaps}
+          </Text>
+        </InfoCard>
+        <InfoCard title='Total Swap Claims'>
+          <Text large color='text.primary'>
+            {blockSummary.numSwapClaims}
+          </Text>
+        </InfoCard>
+        <InfoCard title='Total Open LPs'>
+          <Text large color='text.primary'>
+            {blockSummary.numOpenLps}
+          </Text>
+        </InfoCard>
+        <InfoCard title='Total Closed LPs'>
+          <Text large color='text.primary'>
+            {blockSummary.numClosedLps}
+          </Text>
+        </InfoCard>
+        <InfoCard title='Total Withdrawn LPs'>
+          <Text large color='text.primary'>
+            {blockSummary.numWithdrawnLps}
+          </Text>
+        </InfoCard>
+      </div>
+      <div>
+        <div className='mb-4'>
+          <Text large color='text.primary'>
+            Swaps
+          </Text>
+        </div>
+        <Table>
+          <Table.Thead>
+            <Table.Tr>
```

Comment:
suggestion/todo: at this moment, the `Table` components gets deprecated as it is outdated in designs. The main suggestion, i guess, is to use `TableCell` with grids and subgrids – it proved working very well for other components